R-packages/evalcast/R/manage_cards.R

```diff
@@ -91,3 +91,34 @@ intersect_averagers <- function(cards, grp_vars, avg_vars) {
   }
   cards %>% right_join(averagers_intersected)
 }
+
+#' Scale error measures based on those of a particular forecaster.
+#'
+#' @param score_card score_card like that returned by `evaluate_predictions()`
+#' @param vars vector of column names in `score_card` to normalize
+#' @param denom_forecaster name of forecaster in `score_card$forecaster` column by whose error
+#'   values the remaining forecasters' errors will be scaled
+#' @param err_cols vector of column names in `score_card` that contain error measures.
+#'   Elements of `err_cols` that are not in `vars` will be dropped from the final output to
+#'   avoid a mix of scaled and unscaled measures.
+#'
+#' @return
+#' @export
+scale_by_forecaster <- function(score_card, vars, denom_forecaster, 
+                                err_cols = c("ae", "wis")) {
```

Comment:
Recommend some fast fail errors, if `err_cols` not in the columns of `score_card` or `vars` not in `err_cols`.

Reply:
Implemented failure if `score_cols` (formerly, `vars`) is not a subset of `numeric_err_cols` (formerly `err_cols`).  I don't think it makes sense to validate that `numeric_err_cols` is a subset of `score_card`'s columns since if some are missing, who cares.

Reply:
I would expect that the user who believes that the `numeric_err_cols` is part of the score card they provided would want to know that they are either specifying the wrong columns or using the wrong score card.

Reply:
Obsolete given the removal of that argument